scripts/buildenv-sidecar.sh

```diff
@@ -200,13 +200,19 @@ case "$IMS_ACTION" in
         fi
         ;;
     customize)
-        # copy ca root key into the image before user shell
-        copy_ca_root_key "$IMAGE_ROOT_PARENT/image-root"
-        setup_resolv "$IMAGE_ROOT_PARENT/image-root"
+        # copy ca root key into the image before user shell if running locally
+        if [[ ! -n "${REMOTE_BUILD_NODE}" ]]; then
```

Comment:
Why not use -z instead of ! -n? Seems more straightforward.

Reply:
Because I am a bash scripting newbie...